sdk/state/ingest_test.go

```diff
@@ -820,8 +821,44 @@ func TestChannel_IngestTx_updateState_invalid_initiatorEscrowHasExtraSigner(t *t
+	validResultXDR, err := txbuildtest.BuildResultXDR(true)
+	require.NoError(t, err)
+	resultMetaXDR, err := txbuildtest.BuildResultMetaXDR(xdr.LedgerEntryChanges{
+		xdr.LedgerEntryChange{
+			Type: xdr.LedgerEntryChangeTypeLedgerEntryUpdated,
+			Updated: &xdr.LedgerEntry{
+				Data: xdr.LedgerEntryData{
+					Type: xdr.LedgerEntryTypeAccount,
+					Account: &xdr.AccountEntry{
+						AccountId: xdr.MustAddress(initiatorEscrow.Address()),
+						SeqNum:    102,
+						Signers: []xdr.Signer{
+							{
+								Key: xdr.SignerKey{
+									Type:    xdr.SignerKeyTypeSignerKeyTypeEd25519,
+									Ed25519: xdr.MustAddress("GAKDNXUGEIRGESAXOPUHU4GOWLVYGQFJVHQOGFXKBXDGZ7AKMPPSDDPV").Ed25519,
+								},
```

Comment:
💡 There are some helpers for some XDR types in the xdr package, so you can shorten this a little.
```suggestion
								Key: xdr.MustSigner("GAKDNXUGEIRGESAXOPUHU4GOWLVYGQFJVHQOGFXKBXDGZ7AKMPPSDDPV"),
```

Reply:
https://github.com/stellar/experimental-payment-channels/pull/271/commits/55d2381e67f9855f83edc2a4bb7d529262f52553